--- BUILD/gpsd-2.95/Makefile.am.ldflags	2012-08-31 17:23:12.836530055 +0400
+++ BUILD/gpsd-2.95/Makefile.am	2012-08-31 21:44:09.988528607 +0400
@@ -120,7 +120,11 @@ lcdgps_LDADD = $(LIBM) libgps.la -lm
 gpsmon_SOURCES = gpsmon.c monitor_nmea.c monitor_sirf.c \
 	monitor_italk.c monitor_ubx.c monitor_superstar2.c \
 	monitor_oncore.c monitor_tnt.c
-gpsmon_LDADD = $(LIBM) $(NCURSES_LIBS) libgpsd.la libgps.la -lm $(LIBPTHREAD)  $(LIBUSB_LIBS)
+gpsmon_LDADD = $(LIBM) $(NCURSES_LIBS) libgpsd.la libgps.la -lm -lbluetooth $(LIBPTHREAD)  $(LIBUSB_LIBS)
+gpsmon_la_LDADD = -lbluetooth
+lib_gpsmon_LDADD = -lbluetooth
+
+libgps_json_LDADD = -lbluetooth
 
 #
 # Build gpsdecode
@@ -137,7 +141,7 @@ libgps_VERSION_CURRENT = 19
 libgps_VERSION__REVISION = 0
 libgps_VERSION_AGE = 0
 libgps_VERSION_NUMBER = $(libgps_VERSION_AGE):$(libgps_VERSION__REVISION):$(libgps_VERSION_AGE)
-libgps_la_LDFLAGS = -version-number $(libgps_VERSION_CURRENT):$(libgps_VERSION__REVISION):$(libgps_VERSION_AGE)
+libgps_la_LDFLAGS = -version-number $(libgps_VERSION_CURRENT):$(libgps_VERSION__REVISION):$(libgps_VERSION_AGE) -lbluetooth
 lib_LTLIBRARIES = libgps.la libgpsd.la
 
 libgps_SONAME = $(shell expr $(libgps_VERSION_CURRENT) - $(libgps_VERSION_AGE))
