--- gpsd-3.10/SConstruct.omv~	2014-08-05 20:49:50.906674335 +0200
+++ gpsd-3.10/SConstruct	2014-08-05 20:53:53.061489111 +0200
@@ -1104,7 +1104,7 @@ else:
     ldshared=ldshared.replace('-pie', '')
     python_env.Replace(SHLINKFLAGS=[],
                        LDFLAGS=ldflags,
-                       LINK = ldshared,
+                       LINK = ldshared + " -lpython2.7",
                        SHLIBPREFIX="",
                        SHLIBSUFFIX=so_ext,
                        CPPPATH=[includepy],
