--- gpsd-3.10/libgps_core.c.omv~	2014-08-05 21:17:07.076772770 +0200
+++ gpsd-3.10/libgps_core.c	2014-08-05 21:18:22.534071335 +0200
@@ -18,9 +18,9 @@
 #include "libgps.h"
 #include "gps_json.h"
 
-#ifdef LIBGPS_DEBUG
-int libgps_debuglevel = 0;
+__attribute__((visibility("default"))) __attribute__((used)) int libgps_debuglevel = 0;
 
+#ifdef LIBGPS_DEBUG
 static FILE *debugfp;
 
 void gps_enable_debug(int level, FILE * fp)
--- gpsd-3.10/SConstruct.omv~	2014-08-05 21:22:42.679260821 +0200
+++ gpsd-3.10/SConstruct	2014-08-05 21:25:23.556944928 +0200
@@ -926,7 +926,7 @@ compiled_gpsdlib = Library(env=env,
                            target="gpsd",
                            sources=libgpsd_sources,
                            version=libgpsd_version,
-                           parse_flags=usblibs + rtlibs + bluezlibs)
+                           parse_flags=usblibs + rtlibs + bluezlibs + ['-L. -lgps'])
 
 libraries = [compiled_gpslib, compiled_gpsdlib]
 
